.gitignore

```diff
@@ -16,3 +16,4 @@ venv/
 # Visual Studio Code
 .vscode/*
 !.vscode/tasks.json
+.venv
```

Comment:
Can you remove this? We don't use this file

Reply:
removed and added it to my local ignore list